### DISCUSSION TITLE
Sometimes on Chrome, on trying to connect with Phantom, the browser extension doesn't open up.

### DISCUSSION THREAD

**Discussion by hali-codedistrict** [phantom/docs#127] [Help & Support]
### Summary

Hello, 

We're developing a marketplace that supports multiple Solana based wallets. To support these wallets we're using the Solana Wallet Adapter (v 0.19.20). The problem we're facing is that some of our users (approximately 50%) are reporting that when they click on Phantom (from the Solana Wallet Adapter's UI) the extension does not open up. 

However, some users have been able to fix the the problem by either toggling the extension on/off or by putting it in DevNet and reverting it back to MainNet. 

Thanks in Advance. 

### Steps to Reproduce

The steps to reproduce are really simple: 
1. Sign In and goto settings and then wallets. 
2. Try connecting your wallet (by pressing the add wallet and then selecting Phantom)
3. On some users nothing happens (but when they change their network to devnet and back to mainnet OR toggle the extension on/off, they are able to connect) while other users are able to work on it correctly. 

### Phantom Version

23.17.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @hali-codedistrict, would you mind providing the url where users are experiencing this issue? And have you been able to reproduce yourself and capture any console errors?

**Comment by hali-codedistrict** [comment]
Hello, this url is https://ownersclub.invinciblegg.com/ . The steps to reproduce are: 

- Sign Up / Sign In 
- Goto settings and then wallet 
- click on the add wallet button and then select Phantom 

Nothing happens at this point. The Extension’s pop-up doesn’t open up.

**Reply by adamdelphantom** [reply]
Thank you for providing this! After creating an account and attempting to sign in I get a 'something went wrong error.' Not sure how to proceed. Also could you provide your connection code and the console errors you're seeing when attempting to connect, please? We haven't had any other reports of this issue.

**Reply by hali-codedistrict** [reply]
Can you please make sure you validated your email - this issue normally occurs when you have not validated your email address. 

We don't have any console logs as this issue was reported to us by our users. 

Our wallet connection is handled by the library Solana Wallet Adapted.
